src/components/CheckBoxs.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Text, StyleSheet, View, Pressable } from "react-native";
+import { Ionicons } from '@expo/vector-icons';
+
+export const CheckBox = () => {
+  const [checked, onChange] = React.useState(false);
+
+function onCheckmarkPress() {
+  onChange(!checked);
+}
+
+return (
+  <View style={styles.container}>  
+    <View style= {{flexDirection: "row", justifyContent: "space-between", alignItems: "center",   paddingRight: 19}}>
+      <View style = {{flexDirection: "column", padding: 16}}>
```

Comment:
Can you move these styles to the appropriate StyleSheet? Same as you did with `styles.container` above.